apps/condo/pages/news/[id]/index.tsx

```diff
@@ -363,6 +374,20 @@ const NewsItemCard: React.FC = () => {
                                                 <Button type='primary'>{ResendTitle}</Button>
                                             </Link>
                                         ),
+                                        (isSent && (!newsItem.validBefore || dayjs(newsItem.validBefore) > dayjs()) && newsItemSharings.length === 0) && (
```

Comment:
Is this condition necessary here? `newsItemSharings.length === 0`

Reply:
For example, I want to hide a news item from the resident app, even if I sent it through other channels. Why can't I do that?

Reply:
This is a great point. Right now product solution is to remove that, but i'm not sure how to provide the best ux here